src/models/db/accounts.js

```diff
@@ -43,8 +41,12 @@ function convertToPersistent (data) {
       data.minimum_allowed_balance = Number(data.minimum_allowed_balance)
     }
   } // Otherwise the db defaults minimum_allowed_balance to 0
```

Comment:
No. I added that because @gip wanted a comment for #272 on how the minimum balance is handled in the db.